releases/7.6.0.md

```diff
@@ -105,58 +107,80 @@ The minimum supported version of Python is now 3.10. Python 3.11 is encouraged,
-            "gettext:extract": "./media/node_modules/.bin/vue-gettext-extract",
-            "gettext:compile": "./media/node_modules/.bin/vue-gettext-compile",
-            "ts:check": "./media/node_modules/.bin/vue-tsc --noEmit",
-            "ts:watch": "./media/node_modules/.bin/vue-tsc --watch --noEmit",
-            "start": "./media/node_modules/.bin/cross-env NODE_PATH=./media/node_modules NODE_OPTIONS=--max-old-space-size=2048 ./media/node_modules/.bin/webpack serve --config webpack/webpack.config.dev.js"
+            ...
+    ```
+
+    2. Update the value of `scripts`:
+    ```
+        ...
+        "scripts": {
+            "build_development": "yarn eslint:check && yarn ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack --config ./webpack/webpack.config.dev.js",
+            "build_production": "yarn eslint:check && yarn ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 NODE_ENV=production webpack --config ./webpack/webpack.config.prod.js",
+            "build_test": "yarn eslint:check && yarn ts:check && cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack --config ./webpack/webpack.config.dev.js --env test=true",
+            "eslint:check": "eslint . --ext .vue,.ts",
+            "eslint:watch": "nodemon --watch . --ext ts,vue --exec yarn --silent eslint:check",
+            "eslint:fix": "eslint . --ext .vue,.ts --fix",
+            "gettext:extract": "vue-gettext-extract",
+            "gettext:compile": "vue-gettext-compile",
+            "ts:check": "vue-tsc --noEmit",
+            "ts:watch": "vue-tsc --watch --noEmit",
+            "start": "cross-env NODE_OPTIONS=--max-old-space-size=2048 webpack serve --config ./webpack/webpack.config.dev.js"
         },
         ...
     }
     ```
+    3. Update the dependency references `stable/7.5.0` to `stable/7.6.0`:
+    ```    
+    {
+        "dependencies": {
+            "arches": "archesproject/arches#stable/7.6.0",
+        },
+        "devDependencies": {
+            "arches-dev-dependencies": "archesproject/arches-dev-dependencies#stable/7.6.0"
+        }
+    }
+    ```
 
-4. In settings.py, remove `"compressor"` from `INSTALLED_APPS`.
+4. Move `package.json` from project sub-directory to project root
 
-5. Update your project's JavaScript dependencies:
+    `${project_name}/${project_name}/package.json` -> `${project_name}/package.json`
 
-    1.  In the project's `package.json` file from `stable/7.5.0` to `stable/7.6.0`:
-        ```    
-        {
-            "dependencies": {
-                "arches": "archesproject/arches#stable/7.6.0",
+5. Update `.gitignore`:
+    1. Change `${project_name}/media/node_modules` to `node_modules`
+    2. Change `${project_name}/webpack/webpack-stats.json` to `webpack-stats.json`
+    3. If it exists, remove `${project_name}/webpack/webpack-user-config.js`
+
+6. In settings.py:
```

Comment:
AfS had css with this:

https://github.com/archesproject/arches-for-science/blob/600c7e0516ff894008826ef78297e7701e53040f/afs/media/css/project.css#L2-L5

I know we can't think of everything, but WDYT about mentioning this? Bridge too far?